corpora/sip/metadata/metshdr/SIP5/testCase.xml

```diff
@@ -0,0 +1,108 @@
+      http://earksip.dilcis.eu/schema/SIPVocabularyRecordIDType.xml
+    </dependency>
+  </dependencies>
+  <rules>
+    <rule id="1">
+      <description>mets/metsHdr/altRecordID[@TYPE='SUBMISSIONAGREEMENT'] MAY exist.</description>
+      <error level="INFO">
+        <message>
+          mets/metsHdr/altRecordID[@TYPE='SUBMISSIONAGREEMENT'] does not exist. MAY be: mets/metsHdr/altRecordID[@TYPE='SUBMISSIONAGREEMENT'] exists and its text is a reference to the Submission Agreement associated with the package.
+        </message>
+      </error>
+      <corpusPackages>
+        <package name="SIP_metsHdr_altRecordID_SUBMISSIONAGREEMENT_not_exist" isValid="FALSE">
+          <path>invalid/SIP_metsHdr_altRecordID_SUBMISSIONAGREEMENT_not_exist</path>
+          <description>
+            SIP where the METS.xml file has no mets/metsHdr/altRecordID element with TYPE = 'SUBMISSIONAGREEMENT'.
+          </description>
+        </package>
+        <package name="minimal_SIP_plus_mets_SHOULD_MAY_items" isValid="TRUE">
+          <path>valid/minimal_SIP_plus_mets_SHOULD_MAY_items</path>
+          <description>
+            Minimal SIP that matches all MUST requirements, plus all SHOULD and MAY requirements (both for elements and attributes).
+          </description>
+        </package>
+      </corpusPackages>
+    </rule>
+    <rule id="2">
+      <description>If mets/metsHdr/altRecordID[@TYPE='SUBMISSIONAGREEMENT'] exists then it MUST have a text node.</description>
+      <error level="INFO">
+        <message>
+          mets/metsHdr/altRecordID[@TYPE='SUBMISSIONAGREEMENT'] element has no text node. MUST be: if mets/metsHdr/altRecordID[@TYPE='SUBMISSIONAGREEMENT'] exists then it MUST have text that is a reference to the Submission Agreement associated with the package.
+        </message>
+      </error>
+      <corpusPackages>
+        <package name="SIP_metsHdr_altRecordID_SUBMISSIONAGREEMENT_no_text" isValid="FALSE">
+          <path>invalid/SIP_metsHdr_altRecordID_SUBMISSIONAGREEMENT_no_text</path>
+          <description>
+            A SIP with METS.xml file that:
+            - has mets/metsHdr/altRecordID element with mets/metsHdr/altRecordID/@TYPE = 'SUBMISSIONAGREEMENT'
+            - no text node (i.e. nothing between altRecordID tags)
+            <!-- For example:
+              <altRecordID TYPE="SUBMISSIONAGREEMENT"></altRecordID>).
+            -->
+          </description>
+        </package>
+        <package name="minimal_SIP_plus_mets_SHOULD_MAY_items" isValid="TRUE">
+          <path>valid/minimal_SIP_plus_mets_SHOULD_MAY_items</path>
+          <description>
+            Minimal SIP that matches all MUST requirements, plus all SHOULD and MAY requirements (both for elements and attributes).
+          </description>
+        </package>
+      </corpusPackages>
+    </rule>
+    <rule id="3">
+      <description>mets/metsHdr/altRecordID[@TYPE='SUBMISSIONAGREEMENT'] MAY exist only once.</description>
+      <error level="INFO">
```

Comment:
According to specification there can be only 1 instance of mets/metsHdr/altRecordID[@TYPE='SUBMISSIONAGREEMENT'] 
This should result in an error.